rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "nightly-2023-01-13" 
+channel = "nightly-2023-05-22"
```

Comment:
This is still quite far in the past, any reason for that?

Reply:
yeah, it's the reference bightly parity mentions in the polkadot release